rfcs/53-consistent-hierarchy.md

```diff
@@ -0,0 +1,453 @@
+# Feature Name: `consistent-hierarchy`
```

Comment:
This file shouldn't be here :) Ditto `global-task-pools`.

Reply:
All I did was rebase on main.

Reply:
Hmm, that's weird :(